doc/presentations.rst

```diff
@@ -71,3 +113,4 @@ Videos
 .. _Gael Varoquaux: https://gael-varoquaux.info
 .. _Jake Vanderplas: http://www.vanderplas.com
 .. _Olivier Grisel: https://twitter.com/ogrisel
+.. _Arturo Amor: https://twitter.com/ArturoAmorQ
```

Comment:
As much as I'm honored about the mention, I think it's no longer referenced elsewhere :)